AptosSDK/Src/DTO/RequestsPayloads/ChageTypes/DecodedTableData.cs

```diff
@@ -0,0 +1,18 @@
+using Newtonsoft.Json;
+
+namespace AnkrSDK.Runtime.Aptos.DTO.RequestsPayloads.ChageTypes
+{
+	public class DecodedTableData
+	{
+		[JsonProperty(PropertyName = "key")]
+		// TODO Implement value deserialization
```

Comment:
add tasks for all the todos

Reply:
https://ankrnetwork.atlassian.net/browse/MP-349